specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/bastionHost.json

```diff
@@ -290,6 +262,225 @@
+          }
+        },
+        "x-ms-examples": {
+          "Delete Bastion Shareable Links for the request VMs": {
+            "$ref": "./examples/BastionShareableLinkDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/bastionHosts/{bastionHostName}/getbsl": {
+      "post": {
+        "tags": [
+          "BastionHosts",
+          "BastionShareableLink"
+        ],
+        "operationId": "GetBastionShareableLink",
+        "description": "Return the Bastion Shareable Links for all the VMs specified in the request.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/BastionHostName"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/BastionShareableLinkRequest"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of shareable links for the specified VMs, giving appropriate error messages as needed.",
+            "schema": {
+              "$ref": "#/definitions/BastionShareableLinkListResult"
+            }
+          },
+          "400": {
+            "description": "Bad request. Check the message for details."
+          },
+          "404": {
+            "description": "No access to the bastion or one of the requested virtual machines."
+          }
+        },
+        "x-ms-examples": {
+          "Returns the Bastion Shareable Links for the request VMs": {
+            "$ref": "./examples/BastionShareableLinkGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/bastionHosts/{bastionHostName}/getactivesessions": {
+      "post": {
+        "tags": [
+          "BastionHosts"
+        ],
+        "operationId": "GetActiveSessions",
+        "description": "Returns the list of currently active sessions on the Bastion.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/BastionHostName"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of active sessions on the Bastion.",
+            "schema": {
+              "$ref": "#/definitions/BastionActiveSessionListResult"
+            }
+          },
+          "202": {
```

Comment:
Could you add 'default' to the responses?

Reply:
Done